### PR TITLE
Fix more missing references again.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/devel/MEP/MEP14.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ requires the installation of TeX-specific font packages, for example,
 and can not use TrueType fonts directly.  Unfortunately, despite the
 different semantics for font selection, the same set of font
 properties are used for each.  This is true of both the
-`FontProperties` class and the font-related `rcParams` (which
+`.FontProperties` class and the font-related `rcParams` (which
```
Comment (timhoffm) [inline doc/devel/MEP/MEP14.rst]: `rcParams` Does not resolve.

File: lib/matplotlib/backend_tools.py [hunk 2]
```diff
@@ -1111,7 +1110,7 @@ def trigger(self, *args, **kwargs):
 
 def add_tools_to_manager(toolmanager, tools=default_tools):
     """
-    Add multiple tools to `ToolManager`
+    Add multiple tools to ``.ToolManager``
```
Comment (timhoffm) [inline lib/matplotlib/backend_tools.py]:
```suggestion
    Add multiple tools to `.ToolManager`
```
Either literal or dot-prefixed reference.

Comment (anntzer) [reply lib/matplotlib/backend_tools.py]: fixed

File: doc/devel/MEP/MEP14.rst (hunk 1 above)
Comment (anntzer) [reply doc/devel/MEP/MEP14.rst]: Actually I just realized that
```
`.rcParams`
```
does resolve :)

**Comment by timhoffm** [comment]
I think the point where we could reasonably backport documentation changes to 3.2 or 3.1 has passed because we've had too many documentation changes recently (which is a good thing!).

**Pull request merged**